script/server_bootstrap.sh

```diff
@@ -36,7 +37,7 @@ cd $HOME
 git clone https://github.com/yihangho/SimpleCMS.git --bare
 cd SimpleCMS.git
 mkdir -p $DEPLOYMENT_DIR
-GIT_WORK_TREE=$DEPLOYMENT_DIR git checkout -f master
+GIT_WORK_TREE=$DEPLOYMENT_DIR git checkout -f mcc2015
```

Comment:
I think that depends on your workflow. If you're following [GitHub Flow](https://guides.github.com/introduction/flow/), then deploying to/from master branch is the correct way.